File: petclinic-java-gradle-thymeleaf-tailwind-webpack-htmx-ts/src/main/java/dev/pollito/petclinic_java_gradle_thymeleaf_tailwind_webpack_htmx_ts/HomeController.java
package dev.pollito.petclinic_java_gradle_thymeleaf_tailwind_webpack_htmx_ts;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class HomeController {

    @GetMapping("/")
    public String index() {
        return "home/index";
    }

}

File: petclinic-kt-gradle-thymeleaf-tailwind-webpack-htmx-ts/src/main/kotlin/dev/pollito/petclinic_kt_gradle_thymeleaf_tailwind_webpack_htmx_ts/HomeController.kt
package dev.pollito.petclinic_kt_gradle_thymeleaf_tailwind_webpack_htmx_ts

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class HomeController {

    @GetMapping("/")
    fun index(): String = "home/index"
}
